.gitignore

```
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
+*.log
```

i think here we need use
```suggestion
*.log/* 
```
because *.log will match and ignore all files with the .log extension in any directory within your repository.